### ISSUE TITLE
build fails

### ISSUE THREAD

**Issue by vadimzalunin** [charmie11/CImg-hello-world#2]
Does it require any env variables?

[build]$ cmake ..
CMake Warning at extern/FindCImg.cmake:75 (FIND_PACKAGE):
  By not providing "FindOpenCV.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "OpenCV", but
  CMake did not find one.

  Could not find a package configuration file provided by "OpenCV" with any
  of the following names:

    OpenCVConfig.cmake
    opencv-config.cmake

  Add the installation prefix of "OpenCV" to CMAKE_PREFIX_PATH or set
  "OpenCV_DIR" to a directory containing one of the above files.  If "OpenCV"
  provides a separate development package or SDK, be sure it has been
  installed.
Call Stack (most recent call first):
  CMakeLists.txt:36 (find_package)

-- Could NOT find TIFF (missing:  TIFF_LIBRARY TIFF_INCLUDE_DIR) 
-- A library with BLAS API not found. Please specify library location.
-- LAPACK requires BLAS
-- A library with LAPACK API not found. Please specify library location.
-- A library with BLAS API not found. Please specify library location.
-- checking for module 'OpenEXR'
--   package 'OpenEXR' not found
-- checking for module 'Magick++'
--   package 'Magick++' not found
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
CImg_INCLUDE_DIR (ADVANCED)
   used as include directory in directory /home/vadim/CImg-hello-world

-- Configuring incomplete, errors occurred!
[build]$ echo $?
1

**Comment by charmie11** [comment]
No. As the error message suggested, it requires any of the following files in OpenCV_DIR or CMAKE_MODULE_PATH:
- FindOpenCV.cmake
- OpenCVConfig.cmake
- opencv-config.cmake

CMAKE_MODULE_PATH contains a set of directory names where those CMake related files are saved. Due to some reasons, your system does not have those files in any directory of CMAKE_MODULE_PATH or CMAKE_MODULE_PATH does not contain the directory name.

If you don't need CMake's OpenCV related functions, a simple solution is to comment the 75th line of extern/FindCImg.cmake.
